lms/services/assignment.py

```diff
@@ -210,35 +212,53 @@ def is_member(self, assignment: Assignment, h_userid: str) -> bool:
         )
 
     def get_assignments(
-        self, h_userid: str | None = None, course_id: int | None = None
+        self,
+        instructor_h_userid: str | None = None,
+        course_id: int | None = None,
+        h_userids: list[str] | None = None,
     ) -> Select[tuple[Assignment]]:
         """Get a query to fetch assignments.
 
-        :params: h_userid only return assignments the users is a member of.
-        :params: course_id only return assignments that belong to this course.
+        :param instructor_h_userid: return only assignments where instructor_h_userid is an instructor.
+        :param course_id: only return assignments that belong to this course.
+        :param h_userids: return only assignments where these users are members.
         """
 
-        assignments_query = select(Assignment)
+        query = select(Assignment)
+
+        if instructor_h_userid:
```

Comment:
Same name as in UserService.

On top of the name change, improve the query, the previous filter was filtering assignment where the user was a member of. This changes it to assignments where the user is an instructor.